zest/releaser/release.py

```diff
@@ -251,9 +253,9 @@ def _release(self):
             return
 
         package = self.vcs.name
-        version = self.data['version']
+        tag = self.pypiconfig.tag_format() % ({'version': self.data['version']})
         logger.info("Doing a checkout...")
-        self.vcs.checkout_from_tag(version)
+        self.vcs.checkout_from_tag(tag)
         # ^^^ This changes directory to a temp folder.
         self.data['tagdir'] = os.path.realpath(os.getcwd())
```

Comment:
Ah, I see. It is probably easiest and best to change the tests in `release.txt` to no longer call `_grab_version` and `_check_if_tag_already_exists` individually, but instead to call `prepare`.